_sass/components/_va-playbook.scss

```diff
@@ -0,0 +1,480 @@
+// Adds external icon to all links that begin
+// with http (including https) 
+[href^=http] {
```

Comment:
Should we add another rule to make it not add the external link icon to links that start with https://vets.gov or va.gov?